### PR TITLE
vid2vid   working

### PR DESCRIPTION
**Pull request by nagolinc**
Sort of works, but video is all washed out.

https://user-images.githubusercontent.com/7775917/226836433-3b43676d-f4e2-4d21-87f6-18b42f59f6b1.mp4

### CODE REVIEW

File: scripts/modelscope-text2vid.py [hunk 1]
```diff
@@ -63,13 +71,76 @@ def process(skip_video_creation, ffmpeg_location, ffmpeg_crf, ffmpeg_preset, fps
...
+            print("computing latents")
+            latents = pipe.compute_latents(vd_out).to(device)
+
+            #noise=torch.rand_like(latents)
+            latent_h, latent_w = height // 8, width // 8
+            noise=torch.randn(1, 4, frames, latent_h,
+                                          latent_w).to(device)
+
+
+            latents=latents*(1-img2img_noise)+noise*img2img_noise
```
Comment (nagolinc) [inline scripts/modelscope-text2vid.py:132]: I suspect this line is wrong, but the commented out line below it produces even worse results

File: scripts/modelscope-text2vid.py [hunk 2]
```diff
@@ -63,13 +71,76 @@ def process(skip_video_creation, ffmpeg_location, ffmpeg_crf, ffmpeg_preset, fps
...
+            pipe = savedPipe
+        else:
+            if savedPipe is None:
+                pipe = setup_pipeline()
+            else:
+                pipe = savedPipe
+                savedPipe = None
+
+        device=devices.get_optimal_device()
+        print('device',device)
```
Comment (kabachuha) [inline scripts/modelscope-text2vid.py:87]: please, don't return that pipe stuff :)

**Comment by kabachuha** [comment]
Check @XmYx 's PR. It adds the denoising_strength as a DDIM sampling parameter itself (it affect the denoising scheduler) https://github.com/deforum-art/sd-webui-modelscope-text2video/pull/34

**Comment by kabachuha** [comment]
Overall, it's great that you've got it working at least in some form 🙂
May I add you to the list of collaborators?

**Comment by nagolinc** [comment]
Okay, it's working now.

https://user-images.githubusercontent.com/7775917/226892626-ef02cf35-f51f-4c4b-95bd-36d058c9c2b0.mp4


https://user-images.githubusercontent.com/7775917/226892629-056d6d45-9d68-48de-a67f-ee11c683caf8.mp4

**Comment by nagolinc** [comment]
> Overall, it's great that you've got it working at least in some form 🙂 May I add you to the list of collaborators?

Sounds good

**Review by kabachuha — APPROVED** [review]

**Comment by kabachuha** [comment]
Added to a temp branch, so I could streamline the UI. Will open a pending PR now

And you are invited, check your email

Screenshot 2023-03-22 at 17-26-21 deforum

**Pull request merged**